### ISSUE TITLE
UInt, UByte, ... cannot be wrapped in another AnyVal

### ISSUE THREAD

**Issue by MasseGuillaume** [scala-native/scala-native#635]
Uint is a primitive type we should be able to wrap it in an AnyVal.

Since it's currently implemented with AnyVal it's not possible to wrap it twice.

```scala
class SaFamily(val value: UInt) extends AnyVal
```

```
value class may not wrap another user-defined value class
[error]   class SaFamily(val value: UInt) extends AnyVal
```

**Comment by densh** [comment]
This is a bug in scalac, not Scala Native. I'll keep it open in case we find a different solution for this.

**Comment by densh** [comment]
Fixed in #1426
